src/components/views/editView/leftSidebar/dropCallback.js

```diff
@@ -0,0 +1,73 @@
+import {
```

Comment:
I assume this was added by mistake due to the changes made in the project structure.
I would expect the folder src/components to no longer exist. (It was moved to src/client/components)